hash2curve/src/hash2field/expand_msg/xmd.rs

```diff
@@ -146,12 +147,40 @@ where
     }
 }
 
+/// Error type for [`ExpandMsgXmd`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ExpandMsgXmdError {
+    /// The domain separation tag is empty.
+    DSTError(EmptyDST),
```

Comment:
We shouldn't use the term "error" for an error variant.
```suggestion
    Dst(EmptyDST),
```
I'm also not sure if its really sensible to have a wrapped error type like this instead of just a variant called `EmptyDst`.

Reply:
Fixed in https://github.com/RustCrypto/elliptic-curves/pull/1338/commits/59b7896492da2f48f6751e729da56bb32f177f1b.